scheduler/src/interfaces/datasource-config.ts

```diff
@@ -1,16 +1,6 @@
 export default interface DatasourceConfig {
   id: number
-  protocol: DatasourceProtocol
-  format: object
   trigger: DatasourceTrigger
-  metadata: object
-}
-
-export interface DatasourceProtocol {
-  type: string
-  parameters: {
-    location?: string
-  }
 }
 
 export interface DatasourceTrigger {
```

Comment:
We are having a few files with only a model class. Maybe we can put them to the files where they are used / logically belong to? Only makes sense if these files are rather small ..